pkg/resolve/resolve.go

```diff
@@ -250,6 +250,9 @@ func Resolve(ctx context.Context, cs *params.Run, logger *zap.SugaredLogger, pro
 func pipelineRunsWithSameName(prs []*tektonv1.PipelineRun) error {
 	prNames := map[string]bool{}
 	for _, pr := range prs {
+		if pr.GetName() == "" {
```

Comment:
what about pr with multiple generateName of the same name?

Reply:
PipelineRun with generateName `abc-`  once created in cluster it will have something like
````
abc-tyg
abc-jkj
abc-kjr
````

So i think for that we no need to check the uniqueness

Reply:
yeah but they probably should not call their pipelinerun differently even if it technically works...

just for the fact that `/test PRNAME` would not run otherwise (or run multiple of them)

Reply:
:thinking: 
So considering this should we handle uniqueness for generateName as well ?